### PR TITLE
Fixes formatting for YT on shields page

### LINKED ISSUE #12271: Formatting for Saved Site Exceptions (deleted YT channels) should match ledger table
### Description
After deleting a YT channel from your ledger table, if you go to re-add it under Shields > Saved Site Exceptions > Brave Payments, the YouTube channel is listed, the formatting doesn't match how the payments screen displays it.

### Steps to Reproduce

  1. Visit a YT channel for min time so it displays in your ledger table
  2. Click the trash can icon to remove from ledger table
  3. Navigate to Shields > Saved Site Exceptions > Brave Payments

**Actual result:**
Formatting for site name doesn't display the way it does on ledger:
screen shot 2017-12-13 at 8 55 57 am

**Expected result:**
Formatting should match ledger for readability and in case user had accidentally removes one.

**Reproduces how often:**
Easily

### Brave Version

**about:brave info:**
Brave | 0.19.115
rev | 584e694
Muon | 4.5.25

**Reproducible on current live release:**
yes

### Additional Information
Similar to https://github.com/brave/browser-laptop/issues/12217

### PR DESCRIPTION
**Pull request by NejcZdovc**
## Do not squash commits

Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- [x] Ran `git rebase -i` to squash commits (if needed).
- [x] Tagged reviewers and labelled the pull request [as needed](https://github.com/brave/browser-laptop/wiki/Pull-request-process).

Resolves #12271

Test Plan: defined in #12271

New layout
image

Reviewer Checklist:

Tests

- [ ] Adequate test coverage exists to prevent regressions
- [ ] Tests should be independent and work correctly when run individually or as a suite [ref](https://github.com/brave/browser-laptop/wiki/Code-Guidelines#test-dependencies)
- [ ] New files have MPL2 license header

### CODE REVIEW

**Comment by LaurenWags** [comment]
Verified the following:
1. When YT channel is removed from ledger table via trash can it displays under Shields > Saved Site Exceptions > Brave Payments in a readable format (same as on ledger table without the link)
2. When you click the x next to a YT channel on Shields > Saved Site Exceptions > Brave Payments it is re-added to ledger table when the channel is visited next and displays in a readable format (with link)
3. When you delete this YT channel from ledger table via trash can it displays under Shields > Saved Site Exceptions > Brave Payments again in the same readable format.
4. Multiple YT channels removed from ledger table display under Shields > Saved Site Exceptions > Brave Payments as expected.

**Comment by NejcZdovc** [comment]
added session conversion as well. @bsclifton wdyt, should we go with versions like I did or should we use timestamp check?

File: app/sessionStore.js [hunk 1]
```diff
@@ -830,8 +830,12 @@ module.exports.runPreMigrations = (data) => {
   if (data.lastAppVersion) {
     // Force WidevineCdm to be upgraded when last app version <= 0.18.25
     let runWidevineCleanup = false
+    let formatPublishers = false
 
-    try { runWidevineCleanup = compareVersions(data.lastAppVersion, '0.18.25') < 1 } catch (e) {}
+    try {
+      runWidevineCleanup = compareVersions(data.lastAppVersion, '0.18.25') < 1
+      formatPublishers = compareVersions(data.lastAppVersion, '0.22.0') < 1
```
Comment (NejcZdovc) [inline app/sessionStore.js]: we need to change this to 0.19.118, but is here for testing purposes for now

**Comment by NejcZdovc** [comment]
blocked on #13056

**Comment by NejcZdovc** [comment]
@mrose17 ready

**Pull request merged**